docs/examples/PanelCollapsible.js

```diff
@@ -12,7 +12,11 @@ class Example extends React.Component {
         <Button onClick={ ()=> this.setState({ open: !this.state.open })}>
           click
         </Button>
-        <Panel collapsible expanded={this.state.open}>
+        <Panel
+          collapsible
+          id="collapsible-panel-example"
+          expanded={this.state.open}
+        >
           Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
```

Comment:
panel-body is different because it adds padding for text content. You can't , for instance, use a table or listgroup inside a panel-body component and have it styled correctly. Tis kinda like Card.Layout vs Card.Content